site/_docs/provisioning/index.md

```diff
@@ -5,4 +5,16 @@ title: Getting Started
 permalink: /docs/provisioning/
 ---
 
-Coming soon.
+This is the home of all Provisioning documentation.
+
+If you are new to the platform then the recommended reading is:
+
+- [Prerequisites](/adop-docker-compose/docs/prerequisites/)
+- [Quickstart](/adop-docker-compose/docs/quickstart/)
+- [Architecture Overview](/adop-docker-compose/docs/architecture/)
+
+
+Most common provisioning scenarios:
+
+- [Using the quickstart script](/adop-docker-compose/docs/provisioning/quickstart.md)
+- [Using the adop cli](/adop-docker-compose/docs/provisioning/adop_cli)
```

Comment:
The links dont work for me, but this could be purely because of this not being merged properly.

Reply:
this should show up once it's merged.